pkg/module_manager/module_loader.go

```diff
@@ -92,8 +93,16 @@ func resolveDirEntry(dirPath string, entry os.DirEntry) (string, string, error)
 	// Check if entry is a symlink to a directory.
 	targetPath, err := resolveSymlinkToDir(dirPath, entry)
 	if err != nil {
+		if e, ok := err.(*fs.PathError); ok {
+			if e.Err.Error() == "no such file or directory" {
+				log.Warnf("Symlink target %q does not exist. Ignoring module", dirPath)
```

Comment:
`%q` is handy, but it brings slashes in json logs: "..target \"modules/001-module\" does not exist...". 🥴 
 I prefer to use `'%s'` for logging.

Reply:
This is a problem. JSON will be not valid if a string contains `"`. For example, `{"key": "test '%s' "}` will produce an invalid JSON document if %s is replaced with `"string in quotes"`.